README.md

```diff
@@ -24,11 +26,34 @@ Our [Developer Guide](https://developer.lsst.io/git/git-lfs.html) explains how t
 
 ## Using rc2_subset
 
-To run rc2_subset, first set up a locally cloned version of this package.
+To process RC2 subset data locally, first set up `lsst_distrib` and a locally cloned version of the `rc2_subset` package.
 Then run on the command line:
 
 ```shell=
-NUMPROC=X $RC2_SUBSET_DIR/bin/measureHscRC2Metrics.sh
+$RC2_SUBSET_DIR/bin/run_rc2_subset.sh
 ```
 
-where `NUMPROC=X` sets the desired number of processors to run each `pipetask run` job on.
+This command will not process any data until `--run` is appended to the command.
+Instead, it prints the environment variables that _would_ be used to process RC2 subset data and performs some initial checks that the environment is set up correctly.
+If the environment is not set up correctly, the script will print an error message and exit.
+
+Your locally set up version of the `lsst_distrib` package may not match the current daily EUPS tag, causing the script to exit with an error.
+You can check your currently setup `lsst_distrib` EUPS tag by running: `eups list lsst_distrib -s`.
+To set up `lsst_distrib` for a different tag, you can run: `setup lsst_distrib -t <tag>`.
+Alternatively, to process RC2 subset data using your currently set up EUPS tag instead, set the `RC2_SUBSET_EUPS` environment variable to your preferred value, e.g.:
```

Comment:
This seems a bit confusing. Maybe clarify that if passing an RC2_SUBSET_EUPS value to the shell script you need to also set up the same version of lsst_distrib?

Reply:
Is there a way to specify different values for things like the output collection via kwargs, or do such changes always have to be done directly in the shell script? (I'm thinking about making user collections, for example.)

Reply:
It is also unclear when the setup_rc2_subset.py script is needed. Is this only for the case of a shared repo, and not needed for local processing of a cloned rc2_subset? (Or if it's meant for all cases, maybe some specific examples of how to run it might be helpful.)

Reply:
>This seems a bit confusing. Maybe clarify that if passing an RC2_SUBSET_EUPS value to the shell script you need to also set up the same version of lsst_distrib?

Thanks for going through this on Slack Jeff, much appreciated. After discussion, the error message that is printed to screen when the expected EUPS daily tag does not match the currently set up EUPS tag has been significantly expanded. It now looks like this:
```
ERROR: This script expects lsst_distrib to be set up with a version tag
matching todays date, d_2023_08_06, however, lsst_distrib is currently set
up with the following tags:

    w_2023_30 w_latest current setup 

To resolve this, please either set up lsst_distrib with the expected daily tag:

    setup lsst_distrib -t d_2023_08_06

or change the RC2_SUBSET_EUPS environment variable to match one of the
currently set up lsst_distrib version tags:

    RC2_SUBSET_EUPS=<TAG> ./run_rc2_subset.sh
```

As this allows the user to set `RC2_SUBSET_EUPS` to anything (including, for example, `setup`, which will get things working straight away), I've changed the default output collection name from
```
u/$USER/RC2_subset/$RC2_SUBSET_EUPS
```
to
```
u/$USER/RC2_subset/nightly
```
Not only will this guard against potentially unwanted collection name outputs, but it also makes the default output collection name static, which should aid in modifying tutorial scripts.

In line with this change, I've removed the `RC2_SUBSET_STUB` environment variable option, as that is no longer necessary.

Reply:
> Is there a way to specify different values for things like the output collection via kwargs, or do such changes always have to be done directly in the shell script? (I'm thinking about making user collections, for example.)

Yes, by default, the output will live in the collection:
```
u/$USER/RC2_subset/nightly
```

This can be modified on the command line by setting the `RC2_SUBSET_COLL` environment variable, e.g.:
```
RC2_SUBSET_COLL=u/$USER/my_collection \
./run_rc2_subset.sh
```

For the SLURM batch script usage, the `RC2_SUBSET_COLL` value will be set to `HSC/runs/RC2_subset/$RC2_SUBSET_EUPS`, where `RC2_SUBSET_EUPS` will be the current daily EUPS tag (e.g., `d_2023_08_06` would be assumed today).

Reply:
> It is also unclear when the setup_rc2_subset.py script is needed. Is this only for the case of a shared repo, and not needed for local processing of a cloned rc2_subset? (Or if it's meant for all cases, maybe some specific examples of how to run it might be helpful.)

The `setup_rc2_subset.py` script should only be needed for setting up a new repo for the very first time. Local processing of the cloned `rc2_subset` package does not require running `setup_rc2_subset.py`, as I have modified the data butler on this ticket.

Reply:
Do we do a daily on the day of a weekly?

Reply:
Perhaps we should set `RC2_SUBSET_EUPS` to be `setup` as default? That way, this script will always "just-work" for an end user, and they won't hit the above error message. The error message will only trigger if the user specifically sets `RC2_SUBSET_EUPS` to a given value, as we will during batch submissions. What do you think @jeffcarlin?

Reply:
> Do we do a daily on the day of a weekly?

No, at present we do not. I've spoken with @ktlim about the possibility of adding this tag on weekly days, but it may be non-trivial to implement. I think I can work around this in bash if needs be, but a daily tag would be the principle of least surprise.

Reply:
> Perhaps we should set RC2_SUBSET_EUPS to be `setup` as default?

On consideration, I think this is probably the safest route. I've made the modification and pushed it just now. The non-matching tag error message will now not trigger unless the user specifically requests a given EUPS tag. I think this is a good safety check for those who care about it, but it shouldn't hamper progress for users just trying things out.

I've also modified the README file in line with the above, which hopefully reads better.

Reply:
> Perhaps we should set `RC2_SUBSET_EUPS` to be `setup` as default? That way, this script will always "just-work" for an end user, and they won't hit the above error message. The error message will only trigger if the user specifically sets `RC2_SUBSET_EUPS` to a given value, as we will during batch submissions. What do you think @jeffcarlin?

Yeah, that makes sense, and could potentially avoid confusion from users.